libsolidity/codegen/CompilerContext.cpp

```diff
@@ -313,10 +314,15 @@ void CompilerContext::appendInlineAssembly(
 	ErrorReporter errorReporter(errors);
 	auto scanner = make_shared<Scanner>(CharStream(_assembly), "--CODEGEN--");
 	auto parserResult = assembly::Parser(errorReporter).parse(scanner);
-	if (!parserResult || !errorReporter.errors().empty())
+//	cout << assembly::AsmPrinter()(*parserResult) << endl;
```

Comment:
Stray line.

Reply:
That's one of the two main changes of this PR. I would like to have a way to output the intermediate code, even if analysis is successful.

Reply:
Hm? It is a comment in this PR?

Reply:
Yes and you can temporarily uncomment it if you need the info.

Reply:
Why not put it in an ifdef then?

Reply:
I can do that.